src/lib/parser.ts

```diff
@@ -118,3 +118,66 @@ const parseJcalAlarm = (component: Component): Dictionary => {
 
   return alarmValue;
 }
+
+/**
+ * Parse a freebusy reply
+ *
+ * @param ics string, the vcalendar string containing the vfreebusy component
```

Comment:
Isn't it supposed to be like this?

```suggestion
 * @param {string} ics - The vcalendar string containing the vfreebusy component
```

Reply:
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
done, i'll update the old functions later on in a separate PR